src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+export const TodoInfo:React.FC<Todo> = ({ ...todo }) => (
```

Comment:
не потрібно передавати спредом а потім рестом збирати назад
```suggestion
export const TodoInfo:React.FC<{todo: Todo}> = ({ todo }) => (
```